### DISCUSSION TITLE
Dude why is this happening?

### DISCUSSION THREAD

**Discussion by YourLvurr** [w-okada/voice-changer#1327] [General]
All of a sudden for the past 3 days the voice changer in discord has literally had sped up audio leaving me to sound completely weird. I uninstall and reinstalled, and I even uninstalled discord and reinstalled that too. NOTHING. Still speedy audio whenever I talk, Why is this happening? It was fine before then. I EVEN uninstalled the virtual Audio Cable and then reinstalled that TOO.

**Comment by OnlyKaga** [comment]
I have the same problem and this is becoming annoying. is it because we're using the newest/alpha versions? This is legitimately becoming unusable for discord. I even reset my whole pc just incase I installed something I shouldn't have. Surprise surprise it's still sped up.

**Comment by AnnoyingFox** [comment]
Does it sounds fine out of Discord? For example, setting the output to your speakers instead of virtual audio cable so you can listen to yourself.

**Reply by OnlyKaga** [reply]
I've been talking with Lvurr and It sounds "fine" through the voice changer, all audio ports hooked properly. But all of a sudden when we enter a voice chat it sounds like we're talking in x2 speed. I and him have tried 2 different okada vc versions and it's just the exact same issue.

**Comment by AnnoyingFox** [comment]
If it sounds fine in RVC, and this is happening since 3 days ago, the issue may rely on a recent Discord update. Did you try setting Discord audio settings for voice to Legacy? You can also disable any kind of noise canceling option since RVC output audio won't have noise.

**Reply by OnlyKaga** [reply]
Legacy doesn't change anything when switched to in Audio Subsystem section. So in other words we can't do anything about it?

**Comment by AnnoyingFox** [comment]
Well, RVC output it's fine as you stated. The software haven't changed on itself, there is no auto update or similar option. So if the software is fine, then it may be Discord or VAC.
You can try downloading a free recording software like Audacity and record the VAC output to see if the sound is coming out right. That output is the same that Discord will pick. If still sounds wrong in Discord but fine in Audacity... well, do the math.
The key point right now is to detect where the issue relies.
Have you update anything else that you have in common with your friend besides Discord? Have you update the GPU drivers for example?

**Comment by AnnoyingFox** [comment]
I would need more info. When you open an issue, there is a form to fill with very important information. All that has been deleted and ignored so I don't even know your environment to better help you. I'd like to know which CPU and GPU do you have. Which Windows version. GPU drivers version. RVC version. Voice model type (pth or onnx), which settings are you using in RVC. Which settings are you using in Discord. Which exact virtual audio cable are you using (VAC or vb cable), etc.
There are many variables and with the issue no information of any kind was given so I am not sure from where to start so we can fix it.
